ivy/functional/frontends/torch/nn/functional/convolution_functions.py

```diff
@@ -53,21 +53,19 @@ def _conv_transpose(
 ):
     dims = len(input.shape) - 2
     weight = ivy.permute_dims(weight, axes=(*range(2, dims + 2), 0, 1))
-    for i in range(dims):
-        weight = ivy.flip(weight, axis=i)
     padding, output_padding = map(
         lambda x: [x] * dims if isinstance(x, int) else x, [padding, output_padding]
     )
-    pad_widths = [(weight.shape[i] - 1,) * 2 for i in range(dims)]
-    ret = ivy.conv_general_dilated(
+    pad_widths = [(0,) * 2 for i in range(dims*2)]
+    ret = ivy.conv_general_transpose(
         input,
         weight,
         1,
```

Comment:
since we're using `conv_general_transpose` now, the arguments will should be the same as the input now, so should pass `stride` here now.